host_cleanup.sh

```diff
@@ -43,7 +43,7 @@ ANSIBLE_FORCE_COLOR=true ansible-playbook \
     -b -vvv vm-setup/teardown-playbook.yml
 
 # There was a bug in this file, it may need to be recreated.
-if [[ $OS == "centos" ]]; then
+if [[ $OS == "centos" || $OS == "rhel" ]]; then
```

Comment:
It might be worth breaking this into a separate PR - it's required for my WIP ipv6 work so we clean up the old ipv4 network-scripts

Reply:
https://github.com/metal3-io/metal3-dev-env/pull/95/files